__init__.py

```diff
@@ -234,13 +234,23 @@ def on_volume_get(self, message):
     def on_volume_duck(self, message):
         """ Handle ducking event by setting the output to 0. """
         self.muted = True
+        self.mute_pulseaudio()
         self.set_hardware_volume(0)
 
     def on_volume_unduck(self, message):
         """ Handle ducking event by setting the output to previous value. """
         self.muted = False
+        self.unmute_pulseaudio()
         self.set_hardware_volume(self.volume)
 
+    def mute_pulseaudio(self):
+        """Mutes pulseaudio volume"""
+        call(['pacmd', 'set-sink-mute', '0', 'true'])
```

Comment:
For this and the other commands, consider adding well-named constants to represent the arguments.  For example, what does `0x4b` represent?  What argument are we passing 'true' to?